api/src/main/java/io/strimzi/api/kafka/model/status/ListenerStatus.java

```diff
@@ -54,6 +56,17 @@ public void setType(String type) {
 
     public void setAddresses(List<ListenerAddress> addresses) {
         this.addresses = addresses;
+        if ((addresses == null) || (addresses.isEmpty())) {
+            bootstrap = null;
+        } else {
+            bootstrap = addresses.stream().map(a -> a.getHost() + ":" + a.getPort()).collect(Collectors.joining(","));
+        }
+    }
+
+    @Description("A concatenated list of the bootstrap addresses for this listener.")
```

Comment:
what about using "comma-separated list of hostname:‍port pairs" which is how we already explain what bootstrap means in our doc?

Reply:
I'm all for consistency. Just building that change now and will add a new commit shortly.